rholang/src/main/scala/coop/rchain/rholang/interpreter/matcher/package.scala

```diff
@@ -44,12 +44,12 @@ package object matcher {
 
   private[matcher] def runFirstWithCost[F[_]: Monad: _cost, A](
       f: MatcherMonadT[F, A]
-  ): F[(Cost, Option[(FreeMap, A)])] =
+  ): F[Option[(FreeMap, A)]] =
     for {
       cost <- _cost[F].get
```

Comment:
I think this is no longer used. I'm surprised this compiled :)

Reply:
Already outdated. Made the same mistake.